Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM crystallang/crystal:1.9.2-alpine as Builder
+
+# install packages required to run the representer
+COPY . .
+
+RUN shards install
+
+RUN apk add --no-cache bash coreutils shards yaml-dev musl-dev make
+
+RUN ./bin/build.sh
+
+WORKDIR /lib/ameba
+
+RUN make clean && make
+
+FROM alpine:3.17
```

Comment:
```suggestion
FROM alpine:3.18
```

Reply:
The Crystal lang used in the process uses alpine:3.17: https://github.com/crystal-lang/distribution-scripts/blob/master/docker/alpine.Dockerfile.

So at least in my view would it be good to use same version as that is using.